arm/authorization/client.go

```diff
@@ -0,0 +1,759 @@
+		client.DeleteAtResourceLevelRequestPreparer(),
+		autorest.WithPathParameters(pathParameters),
+		autorest.WithQueryParameters(queryParameters)).Prepare(&http.Request{})
+}
+
+func (client *ManagementLocksClient) DeleteAtResourceLevelRequestPreparer() autorest.Preparer {
+	return autorest.CreatePreparer(
+		autorest.AsJSON(),
+		autorest.AsDelete(),
+		autorest.WithBaseURL(client.BaseUri),
+		autorest.WithPath("/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/{resourceProviderNamespace}/{parentResourcePath}/{resourceType}/{resourceName}/providers/Microsoft.Authorization/locks/{lockName}"))
+}
+
+// CreateOrUpdateAtSubscriptionLevel create or update a management lock at the
+// subscription level.;
+//
+// lockName is the name of lock. parameters is the management lock parameters.
+func (client *ManagementLocksClient) CreateOrUpdateAtSubscriptionLevel(lockName string, parameters ManagementLock) (result ManagementLock, err error) {
+
+	req, err := client.NewCreateOrUpdateAtSubscriptionLevelRequest(lockName, parameters)
+	if err != nil {
+		return result, fmt.Errorf("authorization: Failure preparing ManagementLocksClient.CreateOrUpdateAtSubscriptionLevel request (%v)", err)
+	}
+
+	req, err = autorest.Prepare(
+		req,
+		client.WithAuthorization(),
+		client.WithInspection())
+	if err != nil {
+		return result, fmt.Errorf("authorization: Failure sending ManagementLocksClient.CreateOrUpdateAtSubscriptionLevel request (%v)", err)
+	}
+
+	resp, err := autorest.SendWithSender(client, req)
+	if err != nil {
+		return result, fmt.Errorf("authorization: Failure sending ManagementLocksClient.CreateOrUpdateAtSubscriptionLevel request (%v)", err)
+	}
+
+	result = ManagementLock{}
+	err = autorest.Respond(
+		resp,
+		client.ByInspecting(),
+		autorest.WithErrorUnlessOK(),
+		autorest.ByUnmarshallingJSON(&result),
+		autorest.ByClosing())
+
+	return result, err
+}
+
+func (client *ManagementLocksClient) NewCreateOrUpdateAtSubscriptionLevelRequest(lockName string, parameters ManagementLock) (*http.Request, error) {
+
+	pathParameters := map[string]interface{}{
+		"lockName":       url.QueryEscape(lockName),
+		"subscriptionId": url.QueryEscape(client.SubscriptionId),
+	}
+
+	queryParameters := map[string]interface{}{
+		"api-version": ApiVersion,
+	}
+
+	return autorest.DecoratePreparer(
+		client.CreateOrUpdateAtSubscriptionLevelRequestPreparer(),
+		autorest.WithJSON(parameters),
+		autorest.WithPathParameters(pathParameters),
+		autorest.WithQueryParameters(queryParameters)).Prepare(&http.Request{})
+}
+
+func (client *ManagementLocksClient) CreateOrUpdateAtSubscriptionLevelRequestPreparer() autorest.Preparer {
+	return autorest.CreatePreparer(
+		autorest.AsJSON(),
+		autorest.AsPut(),
+		autorest.WithBaseURL(client.BaseUri),
+		autorest.WithPath("/subscriptions/{subscriptionId}/providers/Microsoft.Authorization/locks/{lockName}"))
+}
+
+// DeleteAtSubscriptionLevel deletes the management lock of a subscription.;
```

Comment:
@brendandixon I have no way of knowing where this `;` comes from as someone who's just reviewing the code. This might as well just can be some mistake in the generation code or somewhere else, I'm just asking.